loopt.py

```diff
@@ -115,14 +115,34 @@ def lookup(self, name):
 
         return self.loops.get(name)
 
+    def lookup_static(self, name):
+        """Look up the srcloc for the named loop in a static function."""
+
+        # CBMC property checking refers to loop K in function FCN as FCN.K,
+        # but the goto-cc --export-file-local-symbols flag renames FCN.K
+        # appearing in a file FILE in a static function FCN as
+        # __CPROVER_file_local_FILE_FCN_LOOP.N
+        keys = [key for key in self.loops.keys() if key.endswith('_'+name)]
+        if not keys:
+            return None
+        if len(keys) != 1:
+            raise UserWarning("Loop name {} matches {} static loop names: {}"
+                              .format(name, len(keys), keys))
+
+        return self.loops.get(keys[0])
+
     def lookup_assertion(self, name):
         """Look up the srcloc for the named loop unwinding assertion."""
 
+        # CBMC refers to the loop K in function FCN as FCN.K and to
+        # the unwinding assertion associated with that loop as
+        # FCN.unwind.K
```

Comment:
Is this guaranteed to remain true? Do we have tests to make sure this doesn't change?

Reply:
Added comment: 

        # Having extracted the loop name from the assertion name, look
        # up the loop trying both its proper name and its mangled
        # name.